server/routes/finalChecks.ts

```diff
@@ -0,0 +1,209 @@
+  reason?: string
+  outOfTimeReasons?: string
+}
+
+export default ({ licenceService, nomisPushService }: { licenceService: LicenceService; nomisPushService }) =>
+  (router, audited, config) => {
+    const standard = createStandardRoutes({
+      formConfig,
+      licenceService,
+      sectionName: 'finalChecks',
+      nomisPushService,
+      config,
+    })
+
+    router.get('/postpone/:bookingId', asyncMiddleware(getPostpone(licenceService)))
+    router.post('/postpone/:bookingId', audited, asyncMiddleware(postPostpone(licenceService, nomisPushService)))
+
+    router.post('/refuse/:bookingId', audited, asyncMiddleware(postRefusal(licenceService, nomisPushService)))
+
+    router.get('/:formName/:bookingId', asyncMiddleware(standard.get))
+    router.post('/:formName/:bookingId', audited, asyncMiddleware(standard.post))
+
+    return router
+  }
+
+function getPostpone(licenceService) {
+  function get(req, res) {
+    const { bookingId, action } = req.params
+
+    const postponeVersion = licenceService.getPostponeVersion(res.locals.licence.licence)
+    const data = firstItem(req.flash('userInput')) || res.locals.licence?.licence?.finalChecks?.postpone || {}
+    const errorObject = firstItem(req.flash('errors')) || {}
+
+    const viewData = {
+      bookingId,
+      data,
+      action,
+      errorObject,
+      version: postponeVersion,
+    }
+
+    if (postponeVersion === '1') {
+      res.render(`finalChecks/postponeV1`, viewData)
+    } else {
+      res.render(`finalChecks/postponeV2`, viewData)
+    }
+  }
+  return get
+}
+
+function postPostpone(licenceService, nomisPushService) {
+  async function post(req, res) {
+    const { bookingId } = req.params
+    const { username } = req.user
+    const originalLicence = res.locals.licence
+    const config = formConfig.postpone
+    const { postRelease } = res.locals
+    const userInput = req.body
+
+    const updatedLicence = await licenceService.update({
+      bookingId,
+      originalLicence,
+      config,
+      userInput,
+      licenceSection: 'finalChecks',
+      formName: 'postpone',
+      postRelease,
+    })
+
+    const errors = validationErrors(updatedLicence.finalChecks.postpone, res)
+
+    if (!isEmpty(errors)) {
+      req.flash('errors', errors)
+      return res.redirect(`/hdc/finalChecks/postpone/${bookingId}`)
+    }
+
+    const { decision } = userInput
+
+    if (decision === 'Yes') {
+      await pushStatus(updatedLicence, bookingId, username)
+    }
+
+    return res.redirect(`/hdc/taskList/${bookingId}`)
+  }
+
+  function validationErrors(formToValidate, res) {
+    return licenceService.validateForm({
+      formResponse: formToValidate,
+      pageConfig: formConfig.postpone,
+      formType: 'postpone',
+    })
+  }
+
+  async function pushStatus(updatedLicence, bookingId, username) {
+    const pushConfig = getIn(formConfig, ['postpone', 'nomisPush'])
+
+    if (getIn(formConfig.postpone, ['pushToNomis']) && pushConfig) {
```

Comment:
There isn't a pushToNomis in the formConfig.postpone object so a a push to Nomis won't occur. 
Should one of these 'getIn' funcs be referencing the server/config.pushToNomis?